### PR TITLE
Typedoc ts 2.0

### PR DESCRIPTION
**Pull request by tmeasday**
Thanks to https://github.com/zakhenry/typedoc/tree/ts-2.0, I got typedoc working w/ TS2.0! (At least for our purposes, AFAICT).

This PR is the comments from https://github.com/apollostack/apollo-client/pull/557 without the code changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "sinon": "^1.17.4",
     "source-map-support": "^0.4.0",
     "tslint": "3.15.1",
-    "typescript": "^2.0.0",
+    "typescript": "2.0.0",
```
Comment (tmeasday) [inline package.json]: Note this has nothing to do with this branch, I'm guessing TS2.0.2 was just released? In any case AC doesn't compile for me.

Comment (stubailo) [inline package.json]: Probably a good idea to lock in the version of TS. Is TS 2 officially out now??

Comment (tmeasday) [inline package.json]: Are you asking me?

I guess not, given https://www.npmjs.com/package/typescript says 1.8.10

**Comment by tmeasday** [comment]
@stubailo please merge away, barring the one thing with TS version

**Pull request merged**